include/swift/Reflection/ReflectionContext.h

```diff
@@ -286,6 +288,100 @@ class ReflectionContext
       return readMachOSections<MachOTraits<8>>(ImageStart);
     return false;
   }
+
+#elif defined(_WIN32)
```

Comment:
Hm, does all of this really need to be conditionalized? In principle, Reflection could be used in a cross-compiler toolchain to inspect binaries for a different target platform from the host.

Reply:
Yes, but, that would be beyond the scope of this PR IMO.  I have been meaning to make the change you just pointed out for a while, but have been stuck with a couple of IRGen issues (the other PRs that I have open).